WorkflowUI/Sources/Screen/AnyScreen/AnyScreen.swift

```diff
@@ -20,23 +20,18 @@ import UIKit
 
 public struct AnyScreen: Screen {
     /// The original screen, retained for debugging
-    internal let wrappedScreen: Screen
-
-    /// Stored getter for the wrapped screen’s view controller description
-    private let _viewControllerDescription: (ViewEnvironment) -> ViewControllerDescription
+    public let wrappedScreen: Screen
```

Comment:
have a slight inclination to put this behind an `_spi` annotation, since it seems like peering into this value should be kind of an exception to typical use.

Reply:
it feels like a standard thing for type erasers to expose the original value. for example Swift's AnyHashable exposes the original value https://developer.apple.com/documentation/swift/anyhashable/base

Reply:
I was gonna say; should we rename this to `base` for consistency?